python/ray/experimental/state.py

```diff
@@ -518,18 +518,28 @@ def task_profiles(self, start=None, end=None, num_tasks=None, fwd=True):
 
         return task_info
 
-    def dump_catapult_trace(self, path, task_info, breakdowns=False):
+    def dump_catapult_trace(self,
+                            path,
+                            task_info,
+                            breakdowns=True,
+                            task_dep=True,
+                            obj_dep=True):
+
         """Dump task profiling information to a file.
 
         This information can be viewed as a timeline of profiling information
         by going to chrome://tracing in the chrome web browser and loading the
         appropriate file.
 
         Args:
-            path: The filepath to dump the profiling information to.
-            task_info: The task info to use to generate the trace.
-            breakdowns: Boolean indicating whether to break down the tasks into
-               more fine-grained segments.
+          path: The filepath to dump the profiling information to.
+          task_info: The task info to use to generate the trace.
```

Comment:
Can you also write that task_info should be the output of `ray.global_state.task_profiles()`?